layers/mldv1_test.go

```diff
@@ -51,7 +54,7 @@ func TestPacketMulticastListenerQueryMessageV1(t *testing.T) {
 		t.Error("Failed to decode packet:", p.ErrorLayer().Error())
 	}
 	checkLayers(p, []gopacket.LayerType{LayerTypeEthernet, LayerTypeIPv6, LayerTypeIPv6HopByHop, LayerTypeICMPv6, LayerTypeMLDv1MulticastListenerQuery}, t)
-	// See https://github.com/google/gopacket/issues/517
+	// See https://github.com/gopacket/gopacket/issues/517
```

Comment:
I don't think this reference works anymore.  Continue to point to `google/gopacket`?

(same comment across all other files that reference this issue)

Reply:
ah yes this needs to be reverted back. can't believe my findall/replace in vscode had negative consequences 😮